<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE TS><TS version="2.0" language="de_DE" sourcelanguage="en">
<context>
    <name>AppInfo</name>
    <message>
        <location filename="store.py" line="414"/>
        <source>Version</source>
        <translation>Version</translation>
    </message>
    <message>
        <location filename="store.py" line="415"/>
        <source>Description</source>
        <translation>Beschreibung</translation>
    </message>
    <message>
        <location filename="store.py" line="416"/>
        <source>Author</source>
        <translation>Autor</translation>
    </message>
    <message>
        <location filename="store.py" line="417"/>
        <source>Firmware</source>
        <translation>Firmware</translation>
    </message>
    <message>
        <location filename="store.py" line="418"/>
        <source>Set</source>
        <translation>Baukasten</translation>
    </message>
    <message>
        <location filename="store.py" line="419"/>
        <source>Model</source>
        <translation>Modell</translation>
    </message>
    <message>
        <location filename="store.py" line="420"/>
        <source>Category</source>
        <translation>Kategorie</translation>
    </message>
    <message>
        <location filename="store.py" line="474"/>
        <source>Inst.</source>
        <translation>Inst.</translation>
    </message>
    <message>
        <location filename="store.py" line="421"/>
        <source>Dependencies</source>
        <translation>Abhängigkeiten</translation>
    </message>
</context>
<context>
    <name>Category</name>
    <message>
        <location filename="store.py" line="45"/>
        <source>System</source>
        <translation>System</translation>
    </message>
    <message>
        <location filename="store.py" line="47"/>
        <source>Models</source>
        <translation>Modelle</translation>
    </message>
    <message>
        <location filename="store.py" line="49"/>
        <source>Tools</source>
        <translation>Werkzeuge</translation>
    </message>
    <message>
        <location filename="store.py" line="53"/>
        <source>Demos</source>
        <translation>Demos</translation>
    </message>
    <message>
        <location filename="store.py" line="59"/>
        <source>&lt;unknown&gt;</source>
        <translation>&lt;unbekannt&gt;</translation>
    </message>
</context>
<context>
    <name>Error</name>
    <message>
        <location filename="store.py" line="282"/>
        <source>Not a valid zip file</source>
        <translation>Keine gültige ZIP-Datei</translation>
    </message>
    <message>
        <location filename="store.py" line="301"/>
        <source>Manifest does not contain a UUID!</source>
        <translation>Manifest enthält keine UUID!</translation>
    </message>
    <message>
        <location filename="store.py" line="314"/>
        <source>Unable to create target dir!</source>
        <translation>Kann Zielverzeichnis nicht erstellen!</translation>
    </message>
    <message>
        <location filename="store.py" line="736"/>
        <source>Error</source>
        <translation>Fehler</translation>
    </message>
    <message>
        <location filename="store.py" line="488"/>
        <source>Update app path mismatch.</source>
        <translation>Aktualisierungspfad stimmt nicht überein.</translation>
    </message>
    <message>
        <location filename="store.py" line="509"/>
        <source>Delete app path not found!</source>
        <translation>Zu löschender Pfad nicht gefunden!</translation>
    </message>
    <message>
        <location filename="store.py" line="515"/>
        <source>Unable to remove app</source>
        <translation>Kann Applikation nicht löschen</translation>
    </message>
    <message>
        <location filename="store.py" line="770"/>
        <source>no version info</source>
        <translation>Keine Versionsinformation</translation>
    </message>
</context>
<context>
    <name>Main</name>
    <message>
        <location filename="store.py" line="838"/>
        <source>Store</source>
        <translation>Store</translation>
    </message>
</context>
<context>
    <name>Menu</name>
    <message>
        <location filename="store.py" line="449"/>
        <source>Install</source>
        <translation>Installieren</translation>
    </message>
    <message>
        <location filename="store.py" line="455"/>
        <source>Uninstall</source>
        <translation>Deinstall.</translation>
    </message>
    <message>
        <location filename="store.py" line="461"/>
        <source>Update</source>
        <translation>Aktualisieren</translation>
    </message>
</context>
<context>
    <name>NetError</name>
    <message>
        <location filename="store.py" line="196"/>
        <source>No error</source>
        <translation>Kein Fehler</translation>
    </message>
    <message>
        <location filename="store.py" line="197"/>
        <source>Connection Refused</source>
        <translation>Verbindung verweigert</translation>
    </message>
    <message>
        <location filename="store.py" line="198"/>
        <source>Server closed connection</source>
        <translation>Server hat Verbindung geschlossen</translation>
    </message>
    <message>
        <location filename="store.py" line="199"/>
        <source>Host not found</source>
        <translation>Server nicht gefunden</translation>
    </message>
    <message>
        <location filename="store.py" line="200"/>
        <source>Connection timed out</source>
        <translation>Zeitüberschreitung in Verbindung</translation>
    </message>
    <message>
        <location filename="store.py" line="201"/>
        <source>Connection cancelled</source>
        <translation>Verbindung abgebrochen</translation>
    </message>
    <message>
        <location filename="store.py" line="202"/>
        <source>SSL handshake failed</source>
        <translation>SSL-Aushandlung fehlgeschlagen</translation>
    </message>
    <message>
        <location filename="store.py" line="203"/>
        <source>Network Failure</source>
        <translation>Netzwerkfehler</translation>
    </message>
    <message>
        <location filename="store.py" line="223"/>
        <source>Network error:</source>
        <translation>Netzwerkfehler:</translation>
    </message>
    <message>
        <location filename="store.py" line="128"/>
        <source>Unknown network error</source>
        <translation type="obsolete">Unbekannter Netzwerkfehler</translation>
    </message>
</context>
<context>
    <name>NewRepo</name>
    <message>
        <location filename="store.py" line="375"/>
        <source>New</source>
        <translation>Neu</translation>
    </message>
    <message>
        <location filename="store.py" line="386"/>
        <source>Name:</source>
        <translation>Name:</translation>
    </message>
    <message>
        <location filename="store.py" line="395"/>
        <source>User:</source>
        <translation>Nutzer:</translation>
    </message>
    <message>
        <location filename="store.py" line="398"/>
        <source>Repository:</source>
        <translation>Repository:</translation>
    </message>
    <message>
        <location filename="store.py" line="391"/>
        <source>Github:</source>
        <translation>Github:</translation>
    </message>
</context>
<context>
    <name>menu</name>
    <message>
        <location filename="store.py" line="842"/>
        <source>New</source>
        <translation>Neu</translation>
    </message>
    <message>
        <location filename="store.py" line="844"/>
        <source>Remove</source>
        <translation>Entfernen</translation>
    </message>
    <message>
        <location filename="store.py" line="850"/>
        <source>ftc</source>
        <translation>ftc</translation>
    </message>
</context>
</TS>
